cypress.config.js

```diff
@@ -1,18 +1,23 @@
 const { defineConfig } = require('cypress');
 
-const { isProd } = require('./utils/config-utils');
+const ENV_DEV = 'development';
+const ENV_PROD = 'production';
+const isDev = process.env.NODE_ENV === ENV_DEV;
+const isProd = process.env.NODE_ENV === ENV_PROD;
 
 module.exports = defineConfig({
   // https://docs.cypress.io/guides/references/configuration#Global
   env: {
-    NODE_ENV: isProd ? 'production' : 'dev',
+    NODE_ENV: process.env.NODE_ENV || ENV_DEV,
+    DEV: isDev,
+    PROD: isProd,
```

Comment:
Keep track of the original `NODE_ENV` and the new `PROD` and `DEV` values (from Vite) in case we need them.